docker/index.yaml

```
@@ -94,6 +94,11 @@ artifactory.algol60.net/csm-docker/stable:
     docker.io/istio/kubectl:
       - 1.5.4
 
+    # This image is used in an init container specified in the loftsman manifest
+    # for the csm-config import job.
+    docker.io/library/alpine:
+      - 3.18
```

Any chance just generic `alpine:3` can be used? I see it does nothing but `mkdir` and `echo`, generic image would be sufficient.

---

Yes, it certainly could. Will that tag just resolve to whatever the latest version of the alpine image is at the time? There are probably lots of other images I could use. I just chose one that I figured would be small enough and perhaps already included so as to not inflate the size of the CSM tar file unnecessarily.

---

@mtupitsyn, could you answer this question? Is it better for me to change this to just alpine:3 as follows?

```suggestion
      - 3
```

---

Yep, `3` is a so called "rolling tag". Currently, rolling tags `3`, `3.18` and `latest` are all pointing to actual latest `3.18.4` at https://hub.docker.com/_/alpine.

---

Thank you, I'll update to use just `3`.